lib/javascript.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     'no-unused-vars': 'error',
     'no-undef': 'error',
     'no-constant-condition': 'error',
+    'no-confusing-arrow': 'error',
```

Comment:
This is not about dependencies.

better exclude this line into another PR to get feedback.

Reply:
Sure, I just saw it as I noticed the `prefer-arrow` plugin is quite old and thought we might be able to remove it.